lib/server.js

```diff
@@ -82,7 +125,6 @@ server.startPrerender = function() {
 		}).catch((err) => {
 			util.log(err);
 			util.log(`Failed to start and/or connect to ${this.browser.name}. Please make sure ${this.browser.name} is running`);
-			this.killBrowser();
```

Comment:
Did you run into any issues with `this.killBrowser()` here before you removed it? If Chrome started but we just couldn't connect, we might still want to try to kill it. I would think all child processes would get properly cleaned up if we exited but just to make sure.

Reply:
To be honest, I can't remember why I removed it, and removed code doesn't have comments :)

I re-added it now, and validated with the tests (how cool!), all looks good.

If I happen to remember why I removed it later, I'll push it into a new PR with some more explanation next time.